src/parse-changelog.ts

```diff
@@ -32,21 +32,35 @@ function isValidChangeCategory(category: string): category is ChangeCategory {
  * @param options.repoUrl - The GitHub repository URL for the current project.
  * @param options.tagPrefix - The prefix used in tags before the version number.
  * @param options.formatter - A custom Markdown formatter to use.
+ * @param options.versionBeforePkgRename - A version string of the package before being renamed.
+ * An optional, which is required only in case of package renamed.
+ * @param options.tagPrefixBeforePkgRename - A tag prefix string of the package before being renamed.
+ * An optional, which is required only in case of package renamed.
```

Comment:
```suggestion
 * @param [options.tagPrefix] - The prefix used in tags before the version number.
 * @param [options.formatter] - A custom Markdown formatter to use.
 * @param [options.versionBeforePkgRename] - A version string of the package before being renamed.
 * An optional, which is required only in case of package renamed.
 * @param [options.tagPrefixBeforePkgRename] - A tag prefix string of the package before being renamed.
 * An optional, which is required only in case of package renamed.
```